### DISCUSSION TITLE
mint nft in sequence

### DISCUSSION THREAD

**Discussion by vishrut66** [metaplex-foundation/metaplex-program-library#939] [Q&A]
hey there , how to mint nft in sequence solana sugarcm3? is there any way to do this??

**Comment by samuelvanderwaal** [comment]
Docs, ser: https://docs.metaplex.com/programs/candy-machine/candy-machine-settings#config-line-settings

**Reply by vishrut66** [reply]
i have tried many times but it was not working. any other way?

**Reply by samuelvanderwaal** [reply]
This is the way to do it, so if it is not working please file a bug report, but it's more likely you're not configuring something correctly.

**Reply by vishrut66** [reply]
can you tell me this is correct or not?  if not then I will report a bug.
![2023-01-08](https://user-images.githubusercontent.com/108527286/211472316-e625acf5-542d-4705-ace9-bd2e214b1880.png)
